dds/idl/v8_generator.cpp

```diff
@@ -243,6 +477,24 @@ bool v8_generator::gen_typedef(AST_Typedef*, UTL_ScopedName* name,
         "  }\n"
         "  return tgt;\n";
     }
+    {
+      Function vtc("copyFromV8", "void");
+      vtc.addArg("src", "const v8::Local<v8::Object>&");
+      vtc.addArg("out", cxx + '&');
+      vtc.endArgs();
+      be_global->impl_ <<
+        "  uint32_t length = 0;\n"
+        "  if (src->IsArray()) {\n"
+        "    length = src->Get(Nan::New(\"length\").ToLocalChecked())->ToObject()->Uint32Value();\n"
+        "  }\n"
+        "  out.length(length);\n"
+        "  for (uint32_t i = 0; i < length; ++i) {\n"
+        "  ";
+      gen_copyfrom("out", "src", elem, "i", true);
+      be_global->impl_ <<
+        "  }\n"
+        "  return;\n";
```

Comment:
Why `return;` here? (Same applies to gen_union and gen_struct.)

Reply:
Again, just following the pattern from what was there before. It maybe helps the reader (of this file) tell when the generated method is complete, but pretty pointless in the output code. I'll get rid of them.